groclient/client.py

```diff
@@ -766,6 +768,12 @@ def get_ancestor(
             True by default. Will perform a lookup() on each ancestor to find name,
             definition, etc. If this option is set to False, only ids of ancestor
             entities will be returned, which makes execution significantly faster.
+        ancestor_level : integer, optional
```

Comment:
add a comment in both methods stating when both distance and ancestor_level are specified, distance takes precedence

Reply:
Awesome will do, I wasn't sure how those two parameters would interact, so thanks for clearing that up.